frontend/src/metabase/selectors/settings.ts

```diff
@@ -90,17 +90,29 @@ export const getDocsUrlForVersion = (
 };
 
 interface UpgradeUrlOpts {
-  utm_media: string;
+  utm_campaign?: string;
+  utm_content: string;
 }
 
 export const getUpgradeUrl = createSelector(
   (state: State) => getPlan(getSetting(state, "token-features")),
   (state: State) => getSetting(state, "active-users-count"),
-  (state: State, opts: UpgradeUrlOpts) => opts.utm_media,
-  (source, count, media) => {
+  (_state: State, utmTags: UpgradeUrlOpts) => utmTags,
+  (plan, count, utmTags) => {
     const url = new URL("https://www.metabase.com/upgrade");
-    url.searchParams.append("utm_media", media);
-    url.searchParams.append("utm_source", source);
+    const searchParams = {
```

Comment:
Upgrade URL change. Please see the issue [description how it has changed](https://github.com/metabase/metabase/pull/47116#issue-2480439695).